ai/prompt/v0.1.0/schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://common.schemas.verida.io/assistant/prompt/v0.1.0/schema.json",
-  "name": "https://common.schemas.verida.io/assistant/prompt/v0.1.0/schema.json",
+  "$id": "https://common.schemas.verida.io/ai/prompt/v0.1.0/schema.json",
+  "name": "https://common.schemas.verida.io/ai/prompt/v0.1.0/schema.json",
```

Comment:
I missed this before, but the `database.name` should be `ai_prompt` to match the schema directory path.